### PR TITLE
Add the support for the last drivers of Raspberry Pi 3 Model B Plus Rev 1.3

### PR DESCRIPTION
**Pull request by geoffreyp**

### CODE REVIEW

File: src/lib/still-camera.ts [hunk 1]
```diff
@@ -40,6 +40,7 @@ export default class StillCamera {
       case 'BCM2711':
       case 'BCM2835 - Pi 3 Model B':
       case 'BCM2835 - Pi 3 Model B+':
+      case 'Raspberry Pi 3 Model B Plus Rev 1.3':
```
Comment (gregnr) [inline src/lib/still-camera.ts:43]: Does `model` really come back with this string on a rev 1.3? Just seems odd that the naming convention would change just for a Pi 3 rev 1.3.

**Review by gregnr — COMMENTED** [review]
Hi @geoffreyp thanks for the PR. I just left one comment I hope you can help answer.

File: src/lib/still-camera.ts (hunk 1 above)
Comment (geoffreyp) [reply src/lib/still-camera.ts:43]: Hello, yes it is the correct string, I just use the name throw by the error 
``Could not determine JPEG signature. Unknown system model '${systemInfo.model}'` `
and it works for me

**Comment by geoffreyp** [comment]
Hello @gregnr !
Yes it is the correct string, I just use the name throw by the error
``Could not determine JPEG signature. Unknown system model '${systemInfo.model}'
and it works for me

**Comment by geoffreyp** [comment]
Hello @gregnr, do you need more information before merge the PR?

**Comment by gregnr** [comment]
Hi @geoffreyp, sorry for the delay and thanks again for the addition. 

We are getting a number of reports like this where the model format has completely changed. Before merging this I would just like to look at the bigger picture to understand what changed.

TBH, we may not even need this check at all, since every model so far has the same JPEG signature. This logic originally existed because people were reporting different JPEG signatures, but this may not have been the right solution. It's possible it should have been based on camera model, not Pi model. 

The biggest blocker right now is getting my hands on some of these Pi's + Cameras and verifying the models and JPEG signatures. Perhaps next steps is to create a tool that makes it easy for developers to report different Pi+Camera+JPEG signature combinations.

**Comment by gregnr** [comment]
I'll take a deeper look today and follow up here.

**Comment by gregnr** [comment]
After further investigation, it appears the Pi model has changed due one of, or the combination of:

- The `systeminformation` dep changed the way Pi model is reported. This happened despite the fact that its major version has not changed (still `v4`). 
- Newer kernel versions report a different model than before (`systeminformation` calls `/proc/cpuinfo` under the hood)

Regardless, we have decided to remove the Pi model check altogether for a couple reasons:

- The check was originally added because the original JPEG signature didn't work for a number of people (including myself at a later date). We guessed that this was because different Pi models had different JPEG encoders and therefore generated different JPEG signatures. We updated to a newer signature and added the model check so that we could return different signatures in the future as people reported them. It's now been over 9 months and every model so far still uses the same signature. Most likely the signature is not based on model after all, and instead based on something else, like camera model or OS version.
- If there are multiple signatures, a better solution in the future would be to simply support multiple signatures, ie. for each MJPEG `data` callback, check all JPEG signatures instead of just one.

This change is available now on [v0.3.4](https://github.com/launchcodedev/pi-camera-connect/releases/tag/v0.3.4).

**Pull request closed without merging**